blinkpy/camera.py

```diff
@@ -483,14 +483,16 @@ def __init__(self, sync):
     @property
     def arm(self):
         """Return camera arm status."""
-        return self.sync.arm
+        return self.motion_enabled
 
     @arm.setter
     def arm(self, value):
         """Set camera arm status."""
-        _LOGGER.warning(
-            "Individual camera motion detection enable/disable for Blink Doorbell is unsupported at this time."
-        )
+        if value:
+            url = f"{self.sync.urls.base_url}/api/v1/accounts/{self.sync.blink.account_id}/networks/{self.sync.network_id}/doorbells/{self.camera_id}/enable"
+        else:
+            url = f"{self.sync.urls.base_url}/api/v1/accounts/{self.sync.blink.account_id}/networks/{self.sync.network_id}/doorbells/{self.camera_id}/disable"
+        return api.http_post(self.sync.blink, url)
```

Comment:
I don't disagree, I was trying to make the smallest change and followed the form factor found here: 
https://github.com/fronzbot/blinkpy/blob/a40011cb74e6e110cd8a5330e3879e4ac7a50c79/blinkpy/camera.py#L450-L454

Maybe once we figure out all of the camera types it would make sense to add each camera type to the API?